packages/core/client/src/schema-component/antd/list/List.style.ts

```diff
@@ -40,8 +40,15 @@ const useStyles = genStyleHook('nb-list', (token) => {
         display: 'flex',
         width: '100%',
         flexDirection: 'column',
-        padding: '4px 5px 0',
-        borderBottom: `1px solid ${token.colorBorderSecondary}`,
+
+        '&:not(:first-child)': {
+          paddingTop: token.paddingContentVertical,
+        },
+
+        '&:not(:last-child)': {
+          paddingBottom: token.paddingContentVertical,
+          borderBottom: `1px solid ${token.colorBorderSecondary}`,
+        },
```

Comment:
建议改成这样，仅使用 :first-child 就可以（具体样式可能需要微调下）
```suggestion
        '&:not(:first-child)': {
          paddingTop: token.paddingContentVertical,
          borderTop: `1px solid ${token.colorBorderSecondary}`,
        },
```

Reply:
实际上 padding 是上下都需要，单独定义一侧不对称，可能会出现贴线的情况。

Reply:
实际上 padding 是上下都需要，否则可能会出现贴线的情况。